src/crypto/PSAOperationalKeystore.cpp

```diff
@@ -209,5 +254,53 @@ void PSAOperationalKeystore::ReleasePendingKeypair()
     mIsPendingKeypairActive = false;
 }
 
+CHIP_ERROR PSAOperationalKeystore::MoveOpKeysFromKVSToITS(FabricIndex fabricIndex) const
+{
+    Crypto::SensitiveDataBuffer<TLV::EstimateStructOverhead(sizeof(uint16_t), P256SerializedKeypair::Capacity())> buf;
+    uint16_t keySize = static_cast<uint16_t>(buf.Capacity());
+
+    if (CHIP_NO_ERROR ==
```

Comment:
This is complex side-effecting logic, with multiple error paths. However, I see no unit tests at all. Please add unit tests that run on POSIX with a POSIX version of PSA API.